### PR TITLE
API Creation

### PR DESCRIPTION
**Pull request by kariabancroft**
- Notes that shall go along with a live code (linked in the notes) for the preparation for shipping service
- I updated the live code per some changes, the old on still is in AdaGold/ada-pets-old if you want to check it out
- TODO; update the live code with show tests

### CODE REVIEW

File: 09-intermediate-rails/10-creating-apis.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+rails new ada-pets --api
+```
+
+For the sake of time we aren't going to create everything from scratch, but you should still know that this is how you will want to start things off when you do this from scratch.
+
+Given the context of our application, we should have a model and controller that reference our main resource, pets. Once you clone this repo, you'll notice that we have these things already created for you!
+
+[https://github.com/AdaGold/ada-pets](https://github.com/AdaGold/ada-pets)
+
+###
```
Comment (droberts-ada) [inline 09-intermediate-rails/10-creating-apis.md]: Did you mean to have a heading here?

File: 09-intermediate-rails/10-creating-apis.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+We are going to approach this application creation from a TDD approach. Right now, you'll see that we have a few tests already created for you. You can run the tests using the usual `rake test`.
+
+You'll notice that we have one basic get route created for pets, and one basic controller action that corresponds with that route.
+
+This error is where we diverge from our "normal" rails approach. Normally, we'd create a view that would correspond with our controller action.  The error should be something like:
```
Comment (droberts-ada) [inline 09-intermediate-rails/10-creating-apis.md]: The wording here is a bit confusing to me - what error are you talking about?

Comment (CheezItMan) [inline 09-intermediate-rails/10-creating-apis.md]: Maybe add a screenshot or copy-paste of the  the initial setup error.

Comment (kariabancroft) [inline 09-intermediate-rails/10-creating-apis.md]: The error is below this block of text

**Pull request merged**